entries/e/ebay.com.json

```diff
@@ -26,7 +26,6 @@
       "gittigidiyor.com",
```

Comment:
Looks like this domain is no longer used. (See [Web Archive](https://web.archive.org/web/20220918233739/http://www.gittigidiyor.com/))
The rest look good :+1: 
```suggestion
```